blockchain_ml_vr.rb

```diff
@@ -0,0 +1,5 @@
+def say_hi
```

Comment:
Have you considered adding some dependency injection to make this more flexible.